crates/pera-core/src/dwallet_mpc/mpc_manager.rs

```diff
@@ -422,9 +422,10 @@ impl DWalletMPCManager {
                             DwalletMPCNetworkKeysStatus::Ready(_)
                         ));
                 if session.is_ready_to_advance() && is_manager_ready {
+                    let session_clone = session.clone();
```

Comment:
Write down why the clone is here in a comment, so no one else will remove it later.

Reply:
You didn't push it